pmm-app/src/shared/components/Elements/Charts/Latency/Latency.tsx

```diff
@@ -110,28 +100,22 @@ export class Latency extends Component<any, LatencyChartState> {
         .attr('cx', `${x(p99)}`)
         .attr('cy', '13px');
     }
-  }
-
-  render() {
-    /* eslint-disable react/no-string-refs */
-    const { className } = this.props;
-    const { tooltip } = this.state;
-
-    return (
-      <>
-        <Tooltip
-          getPopupContainer={() => document.querySelector('#antd') || document.body}
-          placement="left"
-          overlayClassName="latency-tooltip"
-          title={(
-            <pre style={{ backgroundColor: 'transparent', border: 'none' }}>
-              <span>{tooltip}</span>
-            </pre>
-          )}
-        >
-          <div className={`${className || ''} d3-bar-chart-container`} ref="graphContainer" data-tip="" />
-        </Tooltip>
-      </>
-    );
-  }
-}
+  };
+
+  return (
+    <>
+      <Tooltip
+        getPopupContainer={() => document.querySelector('#antd') || document.body}
```

Comment:
Pls find a better way, i.e. no `querySelector` in React.

Reply:
As I understand it's the only way to do it here, the component itself is pretty outdated so I guess we could refactor it and also get rid of a tooltip